apps/DemoApp/Podfile

```diff
@@ -4,7 +4,8 @@ platform :ios, '10.0'
 target 'DemoApp' do
   use_frameworks!
   pod 'Mux-Stats-AVPlayer', :path => '../../'
-  pod 'Mux-Stats-Core', :git => 'https://github.com/muxinc/stats-sdk-objc.git', :branch => 'nidhik/viewer-device-model'
+  pod 'Mux-Stats-Core', :git => 'https://github.com/muxinc/stats-sdk-objc.git', :branch => 'nidhik/ttff'
```

Comment:
Note to self: remember to remove changes to `Podfile` and `Podfile.lock` before merging